cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Actions.hs

```diff
@@ -74,6 +74,19 @@ data NewConstitutionCmd
       , encFilePath :: File () Out
       } deriving Show
 
+-- | Datatype to carry data for the create-info governance action
+-- Will be modified later on to support passing the proposal as a file
+-- (so will be exclusive with the Url and HashSource)
+data NewInfoCmd
+   = NewInfoCmd
+      {   niNetwork :: Ledger.Network
```

Comment:
The alignment inside the curly braces is off. Have a look at: https://github.com/input-output-hk/cardano-node-wiki/wiki/Style-guide

Reply:
Fixed :+1: